### PR TITLE
The Ember Times No. 164 - September 25th 2020

### PR DESCRIPTION
**Pull request by amyrlam**
### Addon/App releases

- [ ] [`ember-tracked-polyfill`](https://github.com/sukima/ember-tracked-polyfill) newly released ([source](https://discordapp.com/channels/480462759797063690/480499624663056390/741699604193607741))
- [ ] [`ember-cli-ui`](https://github.com/rajasegar/ember-cli-ui) newly released ([source 1](https://twitter.com/rajasegar_c/status/1297774711384846336), [source 2](https://twitter.com/rajasegar_c/status/1297775279553646597))
- [ ] [`ember-off-canvas-sidebar`](https://github.com/rajasegar/ember-off-canvas-sidebar) newly released ([source](https://twitter.com/rajasegar_c/status/1301116927247441920))
- [ ] EmberQuest, a geocaching online RPG written in Ember ([source](https://twitter.com/danmonroe/status/1296910382305599488))
  - https://emberquest.danmonroe.dev/
  - https://www.youtube.com/watch?v=Ld1xnQWkqPU
- [x] [ember-concurrency 2.0.0.beta-1](https://twitter.com/m4xm4n/status/1308945677800681473) (🔒 @jaredgalanis )
- [ ] [`ember-apollo-client`](https://github.com/ember-graphql/ember-apollo-client/releases/tag/v3.0.0) v3 released ([source](https://twitter.com/josemarluedke/status/1309571840860327936))

### Articles

- [x] [Async Data and Autotracking in Ember Octane](https://v5.chriskrycho.com/journal/async-data-and-autotracking-in-ember-octane/) (🔒 @chrisrng )
- [x] [EmberJS with REST API](https://jcwebconcepts.net/emberjs-with-rest-api) (🔒 @abhilashlr https://github.com/ember-learn/ember-blog/pull/769)
- [ ] [How Does Ember's Dependency Injection System Work?](https://nullvoxpopuli.com/2020-08-08-how-does-di-work/)
- [x] [Migrating Off of `PromiseProxyMixin` in Ember Octane](https://v5.chriskrycho.com/journal/migrating-off-of-promiseproxymixin-in-ember-octane/) (:lock: @abhilashlr & @amyrlam)
  - Can loop this in to the readers question that was also asked: "preferred ways to replace mixins in Ember Octane" (:lock: @abhilashlr may be able to cover this in a future post - Amy from speaking with locks update: pzuraq and chris krycho could help with replying / moving this forward as well), may relate: https://discuss.emberjs.com/t/migrating-off-of-promiseproxymixin-in-ember-octane/18138
- [ ] [Tracking in the Glimmer VM](https://www.youtube.com/watch?v=BjKERSRpPeI)
- [ ] [Simplify Your Ember App with Domain Components](https://mattfrankjames.com/blog/ember-domain-components/)
- [ ] https://ilyaradchenko.com/ember-context-and-shared-parent-state/
- [x] [Autotracking DX and CS by Chris Krycho](https://v5.chriskrycho.com/journal/autotracking-elegant-dx-via-cutting-edge-cs/)  (🔒 @jaredgalanis )
- [ ] Ember fancy new reactive system from nullvoxpopuli https://twitter.com/nullvoxpopuli/status/1308488990371319809
- [ ] [Switching from Ember-Data to Ember-Orbit](https://medium.com/@mudflye/switching-from-ember-data-to-ember-orbit-82e202248f1c)

### Miscellaneous

- [x] Contributor shout-out! There is a new CLI Guide article - a reference page for all Ember CLI commands! In the writeup, it's good to remind everyone that they can get these with `ember --help` any time. PR https://github.com/ember-learn/cli-guides/pull/196 The new page is available at cli.emberjs.com/release/advanced-use/cli-commands-reference/  (🔒  @ijlee2)
  - Thanks to Tim (https://github.com/fozy81)
  - Ask readers to provide feedback
- [x] Ask for help from the community with joining The Ember Times as a writer (🔒  @ijlee2)
  - TODO: Write a Twitter draft for Melanie Sumner, who can post the tweet via `emberjs`
  - TODO: Finish updating onboarding documentations

### Readers' Questions
  - When could we expect Embroider to be part of Ember-cli and start using it for production apps? (Amy from speaking with locks update: maybe will reach out to person directly. We do not have a RFC for using Embroider in apps, much less an estimate. Let's message direct with Times 161 writeup: https://github.com/emberjs/ember.js/issues/19099.)
  - `ember-data-model-fragments` - Does its release schedule depend on LTS release of Ember Data? (Amy from speaking with locks update: maybe will reach out to person directly and explain it's not an Ember core addon.)
  - A request for visual models (class diagram and activity diagram) for Ember Data (Amy from speaking with locks update: runspired did this for his EmberFest talk, let's connect him with the submitter)

## 📆  Future stories

- https://twitter.com/JohnRodkey/status/1300608539912794113
  > John Rodkey on Twitter: We will be blogging about the move and putting on something other than Medium so there isn't a paywall.  The main reasons, the community, batteries included, and strong conventions.

## 🤔  Questions?

See #support-ember-times on Discord or CONTRIBUTING.

## ✅  Publishing TODOs:

- [ ] Ensure no use of Markdown italic, use `<span style="font-style: italic;">` instead 
- [ ] Check Netlify preview e.g. buttons, to ensure display on emberjs.com/blog is good
- [ ] Crosspost on dev.to, delete any alex, redo any span italics, manually redo Readers Question picture, add Canonical URL 
- [ ] Set up tweet for Monday morning on tweetdeck
- [ ] Post $natural for #news-and-announcements in Discord

### CODE REVIEW

File: source/2020-09-25-the-ember-times-issue-164.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+If you're ready to learn a bit more about how efficient the autotracking system in Ember or Glimmer apps can be, you should head on over to the [post](https://v5.chriskrycho.com/journal/autotracking-elegant-dx-via-cutting-edge-cs/). 
+
+Rest assured, it's a pretty approachable explanation of some complex topics that we as Ember users benefit from in an Octane world. All are encouraged to go check it out!
+
+---
+
+## [Contributors' corner 👏](https://guides.emberjs.com/release/contributing/repositories/)
+
+<p>This week we'd like to thank our siblings for their contributions to Ember and related repositories! 💖</p>
```
Comment (ijlee2) [inline source/2020-09-25-the-ember-times-issue-164.md]:
```suggestion
<p>This week we'd like to thank <a href="https://github.com/amyrlam" rel="noopener noreferrer" target="_blank">Amy Lam (@amyrlam)</a>, <a href="https://github.com/bendemboski" rel="noopener noreferrer" target="_blank">Ben Demboski (@bendemboski)</a>, <a href="https://github.com/bmish" rel="noopener noreferrer" target="_blank">Bryan Mishkin (@bmish)</a>, <a href="https://github.com/bobisjan" rel="noopener noreferrer" target="_blank">Jan Bobisud (@bobisjan)</a>, <a href="https://github.com/chadhietala" rel="noopener noreferrer" target="_blank">Chad Hietala (@chadhietala)</a>, <a href="https://github.com/chancancode" rel="noopener noreferrer" target="_blank">Godfrey Chan (@chancancode)</a>, <a href="https://github.com/chrisrng" rel="noopener noreferrer" target="_blank">Chris Ng (@chrisrng)</a>, <a href="https://github.com/cloke" rel="noopener noreferrer" target="_blank">Cory Loken (@cloke)</a>, <a href="https://github.com/dmytro-krekota" rel="noopener noreferrer" target="_blank">Dmytro Krekota (@dmytro-krekota)</a>, <a href="https://github.com/fozy81" rel="noopener noreferrer" target="_blank">Tim (@fozy81)</a>, <a href="https://github.com/hergaiety" rel="noopener noreferrer" target="_blank">Ava Gaiety Wroten (@hergaiety)</a>, <a href="https://github.com/ijlee2" rel="noopener noreferrer" target="_blank">Isaac Lee (@ijlee2)</a>, <a href="https://github.com/JamesS-M" rel="noopener noreferrer" target="_blank">@JamesS-M</a>, <a href="https://github.com/jaredgalanis" rel="noopener noreferrer" target="_blank">Jared Galanis (@jaredgalanis)</a>, <a href="https://github.com/jenweber" rel="noopener noreferrer" target="_blank">Jen Weber (@jenweber)</a>, <a href="https://github.com/kategengler" rel="noopener noreferrer" target="_blank">Katie Gengler (@kategengler)</a>, <a href="https://github.com/kellyselden" rel="noopener noreferrer" target="_blank">Kelly Selden (@kellyselden)</a>, <a href="https://github.com/kiwiupover" rel="noopener noreferrer" target="_blank">Dave Laird (@kiwiupover)</a>, <a href="https://github.com/knownasilya" rel="noopener noreferrer" target="_blank">Ilya Radchenko (@knownasilya)</a>, <a href="https://github.com/mansona" rel="noopener noreferrer" target="_blank">Chris Manson (@mansona)</a>, <a href="https://github.com/patricklx" rel="noopener noreferrer" target="_blank">@patricklx</a>, <a href="https://github.com/pzuraq" rel="noopener noreferrer" target="_blank">Chris Garrett (@pzuraq)</a>, <a href="https://github.com/raido" rel="noopener noreferrer" target="_blank">Raido Kuli (@raido)</a>, <a href="https://github.com/ro0gr" rel="noopener noreferrer" target="_blank">Ruslan Hrabovyi (@ro0gr)</a>, <a href="https://github.com/rreckonerr" rel="noopener noreferrer" target="_blank">Volodymyr Radchenko (@rreckonerr)</a>, <a href="https://github.com/rwjblue" rel="noopener noreferrer" target="_blank">Robert Jackson (@rwjblue)</a>, <a href="https://github.com/SergeAstapov" rel="noopener noreferrer" target="_blank">Sergey Astapov (@SergeAstapov)</a>, <a href="https://github.com/snewcomer" rel="noopener noreferrer" target="_blank">Scott Newcomer (@snewcomer)</a>, <a href="https://github.com/tempo22" rel="noopener noreferrer" target="_blank">Nicolas Fléron (@tempo22)</a>, and <a href="https://github.com/wycats" rel="noopener noreferrer" target="_blank">Yehuda Katz (@wycats)</a> for their contributions to Ember and related repositories! 💖</p>
```

File: source/2020-09-25-the-ember-times-issue-164.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
...t="_blank">Chris Manson (@mansona)</a>, <a href="https://github.com/patricklx" rel="noopener noreferrer" target="_blank">@patricklx</a>, <a href="https://github.com/pzuraq" rel="noopener noreferrer" target="_blank">Chris Garrett (@pzuraq)</a>, <a href="https://github.com/raido" rel="noopener noreferrer" target="_blank">Raido Kuli (@raido)</a>, <a href="https://github.com/ro0gr" rel="noopener noreferrer" target="_blank">Ruslan Hrabovyi (@ro0gr)</a>, <a href="https://github.com/rreckonerr" rel="noopener noreferrer" target="_blank">Volodymyr Radchenko (@rreckonerr)</a>, <a href="https://github.com/rwjblue" rel="noopener noreferrer" target="_blank">Robert Jackson (@rwjblue)</a>, <a href="https://github.com/SergeAstapov" rel="noopener noreferrer" target="_blank">Sergey Astapov (@SergeAstapov)</a>, <a href="https://github.com/snewcomer" rel="noopener noreferrer" target="_blank">Scott Newcomer (@snewcomer)</a>, <a href="https://github.com/tempo22" rel="noopener noreferrer" target="_blank">Nicolas Fléron (@tempo22)</a>, and <a href="https://github.com/wycats" rel="noopener noreferrer" target="_blank">Yehuda Katz (@wycats)</a> for their contributions to Ember and related repositories! 💖</p>
```
Comment (ijlee2) [inline source/2020-09-25-the-ember-times-issue-164.md]:
```suggestion
<p>This week we'd like to thank <a href="https://github.com/amyrlam" rel="noopener noreferrer" target="_blank">Amy Lam (@amyrlam)</a>, <a href="https://github.com/bendemboski" rel="noopener noreferrer" target="_blank">Ben Demboski (@bendemboski)</a>, <a href="https://github.com/bmish" rel="noopener noreferrer" target="_blank">Bryan Mishkin (@bmish)</a>, <a href="https://github.com/bobisjan" rel="noopener noreferrer" target="_blank">Jan Bobisud (@bobisjan)</a>, <a href="https://github.com/chadhietala" rel="noopener noreferrer" target="_blank">Chad Hietala (@chadhietala)</a>, <a href="https://github.com/chancancode" rel="noopener noreferrer" target="_blank">Godfrey Chan (@chancancode)</a>, <a href="https://github.com/chrisrng" rel="noopener noreferrer" target="_blank">Chris Ng (@chrisrng)</a>, <a href="https://github.com/cloke" rel="noopener noreferrer" target="_blank">Cory Loken (@cloke)</a>, <a href="https://github.com/dmytro-krekota" rel="noopener noreferrer" target="_blank">Dmytro Krekota (@dmytro-krekota)</a>, <a href="https://github.com/fozy81" rel="noopener noreferrer" target="_blank">Tim (@fozy81)</a>, <a href="https://github.com/hergaiety" rel="noopener noreferrer" target="_blank">Ava Gaiety Wroten (@hergaiety)</a>, <a href="https://github.com/ijlee2" rel="noopener noreferrer" target="_blank">Isaac Lee (@ijlee2)</a>, <a href="https://github.com/JamesS-M" rel="noopener noreferrer" target="_blank">@JamesS-M</a>, <a href="https://github.com/jaredgalanis" rel="noopener noreferrer" target="_blank">Jared Galanis (@jaredgalanis)</a>, <a href="https://github.com/jenweber" rel="noopener noreferrer" target="_blank">Jen Weber (@jenweber)</a>, <a href="https://github.com/jfdnc" rel="noopener noreferrer" target="_blank">Jacob (@jfdnc)</a>, <a href="https://github.com/kategengler" rel="noopener noreferrer" target="_blank">Katie Gengler (@kategengler)</a>, <a href="https://github.com/kellyselden" rel="noopener noreferrer" target="_blank">Kelly Selden (@kellyselden)</a>, <a href="https://github.com/kiwiupover" rel="noopener noreferrer" target="_blank">Dave Laird (@kiwiupover)</a>, <a href="https://github.com/knownasilya" rel="noopener noreferrer" target="_blank">Ilya Radchenko (@knownasilya)</a>, <a href="https://github.com/mansona" rel="noopener noreferrer" target="_blank">Chris Manson (@mansona)</a>, <a href="https://github.com/patricklx" rel="noopener noreferrer" target="_blank">@patricklx</a>, <a href="https://github.com/pzuraq" rel="noopener noreferrer" target="_blank">Chris Garrett (@pzuraq)</a>, <a href="https://github.com/raido" rel="noopener noreferrer" target="_blank">Raido Kuli (@raido)</a>, <a href="https://github.com/ro0gr" rel="noopener noreferrer" target="_blank">Ruslan Hrabovyi (@ro0gr)</a>, <a href="https://github.com/rreckonerr" rel="noopener noreferrer" target="_blank">Volodymyr Radchenko (@rreckonerr)</a>, <a href="https://github.com/rwjblue" rel="noopener noreferrer" target="_blank">Robert Jackson (@rwjblue)</a>, <a href="https://github.com/SergeAstapov" rel="noopener noreferrer" target="_blank">Sergey Astapov (@SergeAstapov)</a>, <a href="https://github.com/snewcomer" rel="noopener noreferrer" target="_blank">Scott Newcomer (@snewcomer)</a>, <a href="https://github.com/tempo22" rel="noopener noreferrer" target="_blank">Nicolas Fléron (@tempo22)</a>, and <a href="https://github.com/wycats" rel="noopener noreferrer" target="_blank">Yehuda Katz (@wycats)</a> for their contributions to Ember and related repositories! 💖</p>
```

File: source/2020-09-25-the-ember-times-issue-164.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+<!--alex ignore host-hostess-->
+[Hacktoberfest](https://hacktoberfest.digitalocean.com/) is a celebration of open source open to everyone in the global GitHub community. Whether you’re a developer, student learning to code, event host, or company of any size, you can help drive growth of open source and make positive contributions to an ever-growing community. All backgrounds and skill levels are encouraged to complete the challenge!
+
+Need ideas for Hacktoberfest contributions? Or looking to get started in open source? We suggest working with us on the [next Ember Times issue](https://github.com/ember-learn/ember-blog/pull/778)! As you may know, The Ember Times is the newsletter for the official [Ember Blog](https://blog.emberjs.com/). As writer, you will take on an active role in the Ember community and keep up with the latest events and discoveries. Prior experience in technical writing isn't required, as we will be happy to mentor and pair with you. We also encourage everyone around the globe to join; don't worry if English is your second language (or third, etc.).
```
Comment (ijlee2) [inline source/2020-09-25-the-ember-times-issue-164.md:28]: Perfect transition!

Comment (amyrlam) [reply source/2020-09-25-the-ember-times-issue-164.md:28]: @ijlee2 thanks for all of your help as always, much appreciated!! 💯

Comment (amyrlam) [reply source/2020-09-25-the-ember-times-issue-164.md:28]: teamwork makes the dreamwork!

**Pull request merged**